qoffer-domain/src/main/groovy/life/qbic/portal/portlet/offers/search/SearchOffers.groovy

```diff
@@ -23,12 +23,7 @@ class SearchOffers implements SearchOffersInput{
     }
 
     @Override
-    void searchOffers(SearchCriteria searchCriteria) {
-
-    }
-
-    @Override
-    void downloadOfferAsPdf(String offerId) {
+    void searchOffer(String offerContent) {
```

Comment:
```suggestion
    void searchOffer(String offerContent) {
    //TODO implement
```

Reply:
Did we want to implement the SearchOffer or the SearchCustomer use case?

Reply:
SearchOffer